TechDebtsTracking/core/src/main/java/com/novoda/support/annotations/TechDebt.java

```diff
@@ -0,0 +1,32 @@
+package com.novoda.support.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.SOURCE)
+@Target({ElementType.ANNOTATION_TYPE,
+        ElementType.CONSTRUCTOR,
+        ElementType.FIELD,
+        ElementType.LOCAL_VARIABLE,
+        ElementType.METHOD,
+        ElementType.PACKAGE,
+        ElementType.PARAMETER,
+        ElementType.TYPE})
+public @interface TechDebt {
+
+    enum Size {
+        XSMALL,
```

Comment:
The idea of the annotation is to encourage two things: 

- Tech debts monitoring
- Tech debts addressing. 

Given the second, `Tech debts addressing` `XSMALL` would represent a tech-debt which can be very easily addressed, (5-15 minutes?)so a developer can boyscout it as part of his work if he ever find one of those while working on a certain area.

This was the original idea, of course this PR wouldn't make sense if I am not open to suggestions, so please shout if you have a better idea :D